dissect/target/plugins/apps/container/docker.py

```diff
@@ -48,128 +113,162 @@ class DockerPlugin(Plugin):
 
     __namespace__ = "docker"
 
-    DOCKER_PATH = "/var/lib/docker"
+    def __init__(self, target) -> None:
+        super().__init__(target)
+        self.installs = set(find_installs(target))
 
     def check_compatible(self) -> None:
-        if not self.target.fs.path(self.DOCKER_PATH).exists():
-            raise UnsupportedPluginError("No Docker path found")
+        if not any(self.installs):
+            raise UnsupportedPluginError("No Docker install(s) found.")
 
     @export(record=DockerImageRecord)
     def images(self) -> Iterator[DockerImageRecord]:
         """Returns any pulled docker images on the target system."""
 
-        images_path = f"{self.DOCKER_PATH}/image/overlay2/repositories.json"
-
-        if (fp := self.target.fs.path(images_path)).exists():
-            repositories = json.loads(fp.read_text()).get("Repositories")
-        else:
-            self.target.log.debug(f"No docker images found, file {images_path} does not exist.")
-            return
-
-        for name, tags in repositories.items():
-            for tag, hash in tags.items():
-                image_metadata_path = f"{self.DOCKER_PATH}/image/overlay2/imagedb/content/sha256/{hash.split(':')[-1]}"
-                created = None
-
-                if (fp := self.target.fs.path(image_metadata_path)).exists():
-                    image_metadata = json.loads(fp.read_text())
-                    created = _convert_timestamp(image_metadata.get("created"))
-
-                yield DockerImageRecord(
-                    name=name,
-                    tag=tag,
-                    image_id=_hash_to_image_id(hash),
-                    created=created,
-                    hash=hash,
-                    _target=self.target,
-                )
+        for data_root in self.installs:
+            images_path = f"{data_root}/image/overlay2/repositories.json"
+
+            if (fp := self.target.fs.path(images_path)).exists():
+                repositories = json.loads(fp.read_text()).get("Repositories")
```

Comment:
```suggestion
            if images_path.exists():
                repositories = json.loads(images_path.read_text()).get("Repositories")
```
Combined with the suggestion from @Horofic above.